components/sst_importer/src/sst_importer.rs

```diff
@@ -291,7 +290,20 @@ impl SstImporter {
     ) -> Result<SstImporter> {
         let switcher = ImportModeSwitcher::new(cfg);
         let cached_storage = CacheMap::default();
-        let download_rt = tokio::runtime::Builder::new_current_thread()
+        // We are going to run some background tasks here, (hyper needs to maintain the
+        // connection, the cache map needs gc intervally.) so we must create a
+        // multi-thread runtime, given there isn't blocking, a single thread runtime is
+        // enough.
+        let download_rt = tokio::runtime::Builder::new_multi_thread()
+            .worker_threads(1)
+            .thread_name("sst_import_tiny_runtime")
```

Comment:
What does "tiny" means?

Reply:
I guess perhaps we can rename it "misc_worker" or something?